drama-queen/src/core/usecases/synchronizeData/thunks.ts

```diff
@@ -172,6 +181,87 @@ export const thunks = {
           )
         )
 
+        /*
+         * External special ressources
+         */
+
+        // we sychronize the external ressource only if there is a url for getting them
+        if (externalResourcesUrl) {
+          // get the list of external questionnaires
+          const externalQuestionnaires =
+            await getExternalQuestionnaires().catch((error) => {
+              if (
+                error instanceof AxiosError &&
+                error.response &&
+                [400, 403, 404, 500].includes(error.response.status)
```

Comment:
This check is used at more than 3 places. The good practice is to make it a reusable function (either in a utils file or in this file if it is only used here).